k8s/namespaces/rpe/draft-store-service/draft-store-service.yaml

```diff
@@ -14,10 +14,10 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/draft-store/service:prod-c7a879d-20210807222025   #{"$imagepolicy": "flux-system:draft-store-service"}
       smoketestscron:
-        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025
+        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025   #{"$imagepolicy": "flux-system:service-test"}
       functionaltestscron:
-        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025
+        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025   #{"$imagepolicy": "flux-system:service-test"}
       smoketests:
-        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025
+        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025   #{"$imagepolicy": "flux-system:service-test"}
       functionaltests:
-        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025
+        image: hmctspublic.azurecr.io/draft-store/service-test:prod-c7a879d-20210807222025   #{"$imagepolicy": "flux-system:service-test"}
```

Comment:
This file is missing draft-store? Should be `draft-store-service-test` rather than `service-test` for image policy

Reply:
This looks correct going by the kustomize output.  Do these need changing manually as rerunning the image customisation script still generates the same imagepolicy pattern above 'service-test'? 
Also the kustomize script generates 'service-test-image-policy.yaml/image-repo.yaml' as the policy/repo filenames which is the same for previous others using tests (i.e. reform-scan and cnp plum).

Reply:
As discussed, resolving for now